packages/flutter/test/animation/curves_test.dart

```diff
@@ -105,35 +105,37 @@ void main() {
   });
 
   test('Invalid transform parameter should assert', () {
-    expect(() => const SawTooth(2).transform(-0.0001), throws);
-    expect(() => const SawTooth(2).transform(1.0001), throws);
+    Matcher isAssertionError = const isInstanceOf<AssertionError>();
```

Comment:
I would add "throwsAssertionError" and "throwsFlutterError" to flutter_test just to make these more readable